render_files.py

```diff
@@ -81,15 +81,16 @@ def upload_to_s3(s3_endpoint_url, bucket, aws_access_key_id, aws_secret_access_k
             upload_to_aws(s3_client, file_path, bucket, s3_file_name)
 
 
-def debug_print_upload_to_s3(install_dir):
-    prefix = "dummy_cluster_id"
+def copy_to_local_storage(work_dir, install_dir, cluster_id):
+    os.makedirs(os.path.join(work_dir, cluster_id), exist_ok=True)
     for root, _, files in os.walk(install_dir):
         for file_name in files:
             file_path = os.path.join(root, file_name)
             if file_name == "kubeconfig":
                 file_name = "kubeconfig-noingress"
-            s3_file_name = "{}/{}".format(prefix, file_name)
-            print("Uploading file %s as object %s" % (file_path, s3_file_name))
+            s3_file_name = "/{}/{}/{}".format(work_dir, cluster_id, file_name)
+            print("Copying file %s to %s" % (file_path, s3_file_name))
+            shutil.copyfile(file_path, s3_file_name)
```

Comment:
We don't need the original files, so maybe it would be more efficient to move them rather than copying.  If the source and destination are on the same file system then it would mean changing a pointer rather than reading and writing actual data.

Reply:
Moving the install_dir to a directory named $cluster_id would make it more confusing though. May be symlink to the original files in install_dir?